pkg/table/column.go

```diff
@@ -226,14 +226,28 @@ func convertToIncorrectStringErr(err error, colName string) error {
 	return ErrTruncatedWrongValueForField.FastGen("Incorrect string value '%s' for column '%s'", res.String(), colName)
 }
 
-// handleZeroDatetime handles Timestamp/Datetime/Date zero date and invalid dates.
-// Currently only called from CastValue.
-// returns:
+// handleZeroDatetime handles zero and invalid value for Timestamp/Datetime/Date types.
+// Currently it's only used in CastValue.
+// Returns:
 //
 //	value (possibly adjusted)
 //	boolean; true if break error/warning handling in CastValue and return what was returned from this
 //	error
-func handleZeroDatetime(ec errctx.Context, mode mysql.SQLMode, col *model.ColumnInfo, casted types.Datum, str string, tmIsInvalid bool) (types.Datum, bool, error) {
+func handleZeroDatetime(
```

Comment:
why change it?

if it's only for refactor, do it in another PR, this pr is too large, let's focus on replacing the parquet library